### ISSUE TITLE
Pick a good database visual sofware

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/database-visuals#5]
What we need to have:

- ability to add table title
- ability to create PK, FK
- ability to draw a "relation line"

another complex functionality is not required at this moment.

Main examples:
https://i.stack.imgur.com/rtMUf.png

**Comment by atherdon** [comment]
@khyati0212 can you post that software name that you gave to me?

**Comment by khyati0212** [comment]
DbSchema (https://www.dbschema.com/)

**Comment by atherdon** [comment]
First note - it's not a free software.
Secondnote - i think it's a bit complex. I mean it have a lot of features that we'll never use(i assume).
Can you check some other tools and post it here too?

**Comment by atherdon** [comment]
Check this: https://www.quora.com/What-is-the-best-free-DB-schema-design-tool

**Comment by khyati0212** [comment]
I was looking at the same post. And I think these 2 are good. 

1. SQL Power Architect (Free Download: SQL Power Architect); also available in a paid-for version
2. Oracle SQL Developer Data Modeler (SQL Developer Data Modeler)
Some of these tools are cross-platform, while others are Windows-specific.

**Comment by atherdon** [comment]
Do you want to try a SQL Power Architect? Looks cool and modern.

**Comment by khyati0212** [comment]
Yeah Sure.

**Comment by atherdon** [comment]
ok, let's do this!
Can you try to recreate this schema(https://i.stack.imgur.com/wNjCC.png) and show it to me?
if this tool can create it and it looks good - this can be our choice.

**Comment by khyati0212** [comment]
[TempDB.pdf](https://github.com/ChickenKyiv/database-visuals/files/1798979/TempDB.pdf)
Can you check if this works for you?

**Comment by khyati0212** [comment]
[TempDB.pdf](https://github.com/ChickenKyiv/database-visuals/files/1798995/TempDB.pdf)
This is the prettiest that I can make in that tool. 
And I cannot export it as image. Only PDF and HTML options are available.

**Comment by atherdon** [comment]
Got it. Let's pause this item for now. If we don't find any better - we'll use it.

**Comment by atherdon** [comment]
Can you try same image but on this tool: http://sqldbm.com/

**Comment by atherdon** [comment]
Btw, there is no rush here - we can finish this later. Just for letting you know

**Comment by khyati0212** [comment]
So I have a question. Is this a skill-test?

**Comment by atherdon** [comment]
I think we pass your testing. I mean I read your resume and i'm pretty sure that you're qualified well.
And we started working here :)

**Comment by khyati0212** [comment]
Is there any specific time and task going to assigned for me? Or is it upto me?

**Comment by atherdon** [comment]
No. all is up to you. I mean you have a full freedom here.
Work when you want, spend as much time as you wish. 

Same with the situation if you decide to finish our collaboration - just tell me that you're quit and that it.
And an any results - i'll create a good recommendation letter for you and will try to help to find a full-time, in-office job.

**Comment by atherdon** [comment]
OMG, it's night at your timezone right?

**Comment by khyati0212** [comment]
Thank you for the information. Its 7pm here. So I was hoping if you can give me some tasks for day and I will do them when I get time during the day and update you as we are not in same timezone. Will that work?

**Comment by atherdon** [comment]
Yes, it'll work perfectly!

When I'll complete other tasks at this repository - I'll assign it to you(as i did it here). And add badge - "help wanted". You'll be able to review that tasks, ask questions if something not clear - as we did here.
Btw, we still will need to find a right tool for working with schemes/diagrams, so when you'll have time - please try that second tool.

I'm working at different time zones, so it's not easy to catch me sometime, but if you would ask a question at this issues(and use `@atherdon` mention) - i'll receive notifications and reply when i'll have time.
I also have a bad habit - I can go into a stealth mode sometimes. When I'm working on something complex and be over distracted - I shut off all my communication channels, so if i'm not replying - then i'm working on something. Just let you know

**Comment by khyati0212** [comment]
Oh this sounds good. I can mostly work from 10am - 7pm and than 12am to 1am according to my timezone. So I will update you soon.

**Comment by atherdon** [comment]
Good! thank you for your time - and welcome to the team.

Btw, that part with Recipe <--> Ingredient schema(second image that you've recreating) - we'll use it soon.

**Comment by khyati0212** [comment]
I will be doing coding as well. Right?

**Comment by atherdon** [comment]
if you want - we can start coding from tomorrow :) I just want to intro you to the project. And it's a great approach to create a specification/diagram of the database before starting to work and test relations. Can save a lot of time.

**Comment by khyati0212** [comment]
I understand. I was just curious to know that is why I asked. Anyways l will try the second tool and that you can let me know other tasks.

**Comment by atherdon** [comment]
it's ok - feel free and ask me anything - this is my main goal to create a good work environment for each team member. 

Second note - this work must be fun :)
So if for example, you'll be bored with this diagrams - you tell me and we'll pick another task for you.
Everything is negotiable and related to a comfortable teamwork.

**Comment by atherdon** [comment]
Noted. Will think about giving you a coding-backend related task asap.

**Comment by khyati0212** [comment]
Thank you

**Comment by khyati0212** [comment]
I might not be very fast but I will make sure I do good work for you.

**Comment by khyati0212** [comment]
I am glad to be here.

**Comment by atherdon** [comment]
it's a pleasure for me too :) Must go and grab some meal - buzz me when you'll need my help and thanks again for your time

**Comment by khyati0212** [comment]
Sure. Its dinner time for me too. :)

**Comment by atherdon** [comment]
https://sqldbm.com/Project/SQLServer/Share/9p91gc4WFt5THDzeqjNK6Q

**Comment by khyati0212** [comment]
https://sqldbm.com/Project/MySQL/Edit/p11053/

**Comment by atherdon** [comment]
@khyati0212 can you generate a shareable link? i didn't have an account at sqlDBM

**Comment by khyati0212** [comment]
Here it is:  https://sqldbm.com/Project/MySQL/Share/u4h6KCnmMhvVzvwbeEapmQ

**Comment by atherdon** [comment]
I like it, but I assume we have a duplicated fields/
Btw, I think we can close this task(when you'll update the scheme) - because for my opinion - this software is a good tool for us, right?

**Comment by khyati0212** [comment]
Yes. It is good tool. I agree.

**Comment by atherdon** [comment]
#7
